contrib/cirrus/integration_test.sh

```diff
@@ -29,7 +32,6 @@ export CONMON_BINARY=/usr/libexec/crio/conmon
 export PAUSE_BINARY=/usr/libexec/crio/pause
 export CRIO_CNI_PLUGIN=/usr/libexec/cni
 
-
 echo "Executing cri-o integration tests (typical 10 - 20 min)"
 cd "$GOSRC"
-timeout --foreground --kill-after=45m 30m ./test/test_runner.sh
+timeout --foreground --kill-after=90m 60m ./test/test_runner.sh
```

Comment:
This will send SIGTERM then wait 30-minutes before sending SIGKILL.  Probably want that to be tighter, in fact the 15-minutes I used is probably too long.  IMHO, if the tests can't shut themselves down in less that 5 minutes or so, they probably never will (or it's not worth waiting).  Best get that problem in front of an engineer sooner rather than later (IMHO).